python/ql/src/Security/CWE-798/HardcodedCredentials.ql

```diff
@@ -80,6 +81,8 @@ class HardcodedValueSource extends DataFlow::Node {
 
 class CredentialSink extends DataFlow::Node {
   CredentialSink() {
+    this = ModelOutput::getASinkNode("credentials-hardcoded").asSink()
```

Comment:
From looking through our current support for this in [JS](https://github.com/github/codeql/blob/c2942b37a7ce8f870bd70f1a400734de3d165dfa/javascript/ql/lib/semmle/javascript/frameworks/Credentials.qll#L32) and CredentialsSink() { exists(string s | s.matches("credentials-%") | sinkNode(this, s)) }, and the [current modeling of credentials sinks](https://github.com/search?q=repo%3Agithub%2Fcodeql+%22credentials-%22++language%3AYAML&type=code&l=YAML), it looks like the other languages identify the credentials type as well, so use sink-types such as `"credentials-password"` or `"credentials-username"`.

So we need something like 
```suggestion
    exists(string s | s.matches("credentials-%") |
      // Actual sink-type will be things like `credentials-password` or `credentials-username`
      this = ModelOutput::getASinkNode(s).asSink()
    )
```

Reply:
Makes complete sense! I will be sending in some models soon and best to be consistent across languages